sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-files/e-cert-response-record.ts

```diff
@@ -0,0 +1,57 @@
+import { ECertResponseRecordIdentification } from "./e-cert-response-record-identification";
+
+/**
+ * Parsing E-Certs feedback records to get the document number and associated
+ * error codes
+ * Please note that the numbers below (e.g. line.substr(25, 2))
+ * represents the position of the information in a fixed text file format.
+ * The documentation about it is available on the document
+ * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
+ */
+export class ECertResponseRecord extends ECertResponseRecordIdentification {
+  constructor(line: string, lineNumber: number) {
+    super(line, lineNumber);
+  }
+
+  /**
+   * document Number Financial document number associated with this disbursement.
```

Comment:
It seems that this comment must be reviewed. Maybe only "Financial document number associated with this disbursement." would be enough.